source/blog/openshift-openId-integration-with-engine-sso.html.md

```diff
@@ -1,55 +1,57 @@
 ---
-title: Configuring Kibana/Elastic Search/Open Shift backend to authenticate with engine sso
+title: Integrating Kibana/ Elastic Search on top of Openshift with oVirt Engine SSO
```

Comment:
It is called "Elasticsearch", all one word, not "Elastic Search" or other two word formulations.